lib/commands/element.js

```diff
@@ -45,8 +45,13 @@ commands.getRect = async function (el) {
 commands.getLocation = async function (el) {
   el = el.ELEMENT ? el.ELEMENT : el;
   if (this.isWebContext()) {
-    let atomsElement = await this.useAtomsElement(el);
-    return await this.executeAtom('get_top_left_coordinates', [atomsElement]);
+    const atomsElement = await this.useAtomsElement(el);
+    const script = "return [document.body.scrollLeft, document.body.scrollTop];";
+    const [xOffset, yOffset] = await this.execute(script);
+    let loc = await this.executeAtom('get_top_left_coordinates', [atomsElement]);
```

Comment:
_const_ does not make the value immutable. See http://exploringjs.com/es6/ch_variables.html#ch_variables, p. 9.3.1 for the reference ;)